test/acceptance/features/steps/steps.py

```diff
@@ -420,13 +419,17 @@ def invalid_sbr_is_applied(context):
         json_path = "{.metadata.resourceVersion}"
         rv = sbr.get_servicebinding_info_by_jsonpath(context.sbr_name, context.namespace.name, json_path)
         context.resource_version = rv
-    context.expected_error = sbr.attempt_to_create(context.text)
+    context.expected_error = sbr.attempt_to_create_invalid(context.text, context.namespace.name)
 
 
+@then(u'Error message is thrown')
 @then(u'Error message "{err_msg}" is thrown')
-def validate_error(context, err_msg):
-    search = re.search(rf'.*{err_msg}.*', context.expected_error)
-    assert search is not None, f"Actual error: '{context.expected_error}', Expected error: '{err_msg}'"
+def validate_error(context, err_msg=None):
+    if err_msg is None:
+        assert context.expected_error is not None, f"No error message should happen, but was {err_msg}"
```

Comment:
The assertion message is reverted
```suggestion
        assert context.expected_error is not None, f"Any error message should happen, but was {context.expected_error}"
```

Reply:
good catch.

this is the case where we did not specify the error message, but some error should happen - if assertion fails, then the `expected_error` is `None`, IMHO not need to put that in the assertion error message. How about this:

```
assert context.expected_error is not None, f"An error message should happen"
```

Reply:
I agree, we don't need to mention the `None` in the message :)